oauth2client/contrib/django_util/__init__.py

```diff
@@ -210,6 +264,20 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """Initializes setting to control whether to use a Django model to store
+    OAuth2 credentials in. By default, the credentials will be stored in the
+    session."""
```

Comment:
Please add a `Returns` section here, so we know what callers are expecting to be returned from this function.